app/jobs/slack_event/admins_job.rb

```diff
@@ -0,0 +1,64 @@
+#
+# This comes through the slash commands API which has a
+# slightly different payload than the events API
+#
+# Example payload:
+#
+# {
+#   "token": "xxxx",
+#   "team_id": "TCF0ZD3LL",
+#   "team_domain": "railslink-dev",
+#   "channel_id": "xxxx",
+#   "channel_name": "xxxx",
+#   "user_id": "xxxx",
+#   "user_name": "xxxx",
+#   "command": "/admins",
+#   "text": "",
+#   "response_url": "https://hooks.slack.com/commands/TCF0ZD3LL/640895425681/GDK2vHvOpxpvFrpRJnrvxiHS",
+#   "trigger_id": "xxxx",
+#   "slack": {
+#     "event": {
+#       "type": "admins"
+#     }
+#   },
+#   "controller": "slack",
+#   "action": "event"
+# }
+#
+
+class SlackEvent::AdminsJob < ApplicationJob
+
+  attr_reader :options
+  private :options
+
+  def perform(options = {})
+    @options = options.with_indifferent_access
+    message = {
+      text: admin_text,
+      response_type: "in_channel"
+    }
+    HTTParty.post(URI::parse(@options[:response_url]), body: message.to_json)
+  end
+
+  private
+
+  # Return the text message for the list of admins.
+  #
+  # The text message should not be wrapped as that will break Slack's automatic
+  # wrapping on presentation.
+  #
+  # Returns String (or nil)
+  def admin_text
+  <<~END_OF_TEXT
+    Admins and their contact information:
+    #{admin_string}
+    END_OF_TEXT
+  end
+
+  def admin_string
+    admins_string = ""
+    SlackUser.admins.shuffle.each { |admin| admins_string << "#{admin.first_name} #{admin.last_name} | email: #{admin.email} | timezone: #{admin.tz}\n" }
```

Comment:
So - couple of things about this: initially we had talked about a total random sort so that the same people were not always at the top of the list and getting the requests (same problem with alphabetical).  Also if one of the admins who needs to be reported on is also one of our most active admins, they will always be at the top of the list.

Reply:
True. I think the other side of the coin was we wanted active admins to be more likely to be contacted since reaching out to a non active one would result in no response maybe.  But you make a good point...

Maybe leave this for now.. and combine the two into a method they both can use in a later patch?

Reply:
Maybe in the message we could put their last active date?

Reply:
Ooooh, I really like that idea!

Reply:
K.  I'll put that in place when I get home tonight, stay tuned.

Reply:
Pushed this fix just now @phallstrom